src/Turbo/Doctrine/BroadcastListener.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Kévin Dunglas <kevin@dunglas.fr>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\Turbo\Doctrine;
+
+use Doctrine\Common\EventArgs;
+use Doctrine\ORM\Event\OnFlushEventArgs;
+use Symfony\Contracts\Service\ResetInterface;
+use Symfony\UX\Turbo\Broadcast;
+use Symfony\UX\Turbo\Broadcaster\BroadcasterInterface;
+
+/**
+ * Detects changes made from Doctrine entities and broadcasts updates to the Mercure hub.
+ *
+ * @author Kévin Dunglas <kevin@dunglas.fr>
+ *
+ * @see https://github.com/api-platform/core/blob/master/src/Bridge/Doctrine/EventListener/PublishMercureUpdatesListener.php Adapted from API Platform.
+ *
+ * @todo backport MongoDB support
```

Comment:
what does this mean? when will it be done?

Reply:
The API Platform code I adapted here also supports Doctrine MongoDB ODM. We could add support for Mongo later.